### DISCUSSION TITLE
Clarification of using custom generators

### DISCUSSION THREAD

**Discussion by hansbala** [wailsapp/wails#3959] [General]
I've been playing around with Wails recently in one of my monorepo applications and I love it so far!

The issue I've been experiencing is that I use protos as my data exchange format to web apps too and those use custom generated proto definitions. I would have liked the ability to tell Wails, "Hey! Don't generate TypeScript types but instead use these pre-generated ones". 

I did see a few plans on extending the generator in v3 but couldn't find any information on whether Wails would support custom type generation.

**Comment by leaanthony** [comment]
Totally open to this though I'm not sure what that would mean in terms of the server side. Would it need to open a port?

**Reply by hansbala** [reply]
@leaanthony maybe this was meant for another thread? Don't think this has to do with server / port?

**Reply by leaanthony** [reply]
Haha absolutely. GitHub mobile is 💩
